napari_svg/layer_to_xml.py

```diff
@@ -178,16 +178,20 @@ def points_to_xml(data, meta):
     else:
         face_color = np.ones((data.shape[0], 4))
 
-    if 'edge_color' in meta:
-        edge_color = meta['edge_color']
+    if 'border_color' in meta:
+        stroke_color = meta['border_color']
```

Comment:
Since `stroke-width` is the SVG name, I changed the variable name to reflect that instead of edge/border.